docs/source/scripts_and_their_functions.rst

```diff
@@ -23,7 +23,7 @@ into a sequence of EBU-TT Live documents using natural language processing. Use
 The default carriage mechanism is WebSocket, so you will need to listen to
 ``ws://127.0.0.1:9000``. Conveniently, we've created an HTML page that does just
 that. After you launch the Simple Producer, open ``docs/build/ui/test/index.html``
-or the `current release pre-built page <../ui/test>`_ in your
+or the `current release pre-built page <../ui/test/index.html>`_ in your
```

Comment:
as just discussed, the link doesn't go to the current release pre-built page anyway, since it's a relative link - it will just go to the current built version wherever that is. Probably should add this hyperlink around the link text earlier in the sentence, and change this hyperlink to be hard coded to the ebu.github.io one.